### ISSUE TITLE
Who can help me fix the problem of 'Net Crash' ?

### ISSUE THREAD

**Issue by FightingZhen** [tensorflow/tensorflow#9039]
I am training a convolutional network, but suddenly that happened. I retry for 3 times, same scene happened. The network looks as if it crashed ! Who can help me fix this problem?
The scene looks like the following:
Epoch:100 training_accuracy:0.975 validation_accuracy:0.94242
Epoch:101 training_accuracy:0.075 validation_accuracy:0.103736
Epoch:102 training_accuracy:0.075 validation_accuracy:0.103736

**Comment by derekhh** [comment]
I've seen some similar behavior during my training and my issue was similar to #1122. Can you try tuning your `weight_decay` parameters in the batch normalization layer and see if that helps?

**Comment by asimshankar** [comment]
There isn't enough information in the report to be able to help. I'd suggest posting to [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) with more details as there is a larger community that monitors that and might be able to help.
